share/picongpu/pypicongpu/template/include/picongpu/param/incidentField.param.mustache

```diff
@@ -135,7 +135,7 @@ namespace picongpu::fields::incidentField
          */
         static constexpr float_64 W0_SI = {{{waist_si}}};
 
-        /** The laser pulse will be initialized PULSE_INIT times of the PULSE_LENGTH
+        /** The laser pulse will be initialized PULSE_INIT times of the PULSE_DURATION

```

Comment:
not just half of it?

Reply:
No, it is the total initialization time measured in `PULSE_DURATION`s

Reply:
You are right :+1: